playjson/src/test/scala/io/finch/playjson/Bar.scala

```diff
@@ -0,0 +1,3 @@
+package io.finch.playjson
+
+case class Bar(x: Int, y: Boolean)
```

Comment:
I guess you should provide `Writes[Bar]` and `Reads[Bar]` to make this compile. Try this:

```scala
import play.api.libs.json._

case class Bar(...)
object Bar {
  implicit val barWrites = Json.writes[Bar]
  implicit val barReads = Json.reads[Bar]
}
```